adoc/admin-security-role-management.adoc

```diff
@@ -226,3 +226,17 @@ roleRef:
 
 
 ====
+
+[IMPORTANT]
+====
+When creating new Roles, ClusterRoles, and bindings, it is important to keep in
+mind the `Principle of Least Privilege`: define rules such that the entity
+bound to the Role or ClusterRole has the minimum amount of permissions needed
+to fulfill their purpose and no more. For instance, granting the `admin`
+ClusterRole to most users is most likely unnecessary, as a reduced-scope role
+could meet the user's requirements. This helps reduce the attack surface if a
+user with access to the cluster is compromised.
```

Comment:
This is even more minor nit picking, but what about "an account" rather than "a user".  I think we usually consider "user" to be synonymous with "human".  We reference "service account" elsewhere in the documentation, and also talk about "user authentication."  So this may need a short discussion somewhere to establish a standard?  Or I may need to eat some lunch and quit overthinking things. :D

Reply:
I think "account" probably makes the most sense here